samcli/lib/init/cli_template_modifier.py

```diff
@@ -0,0 +1,211 @@
+"""
```

Comment:
Sorry for bugging this again, but this file and all classes are only working for enabling xray tracing. But they have very generic names, it sounds like these can be used to modify template but it is not actually. Please either extract generic parts into classes and files which will have generic names, otherwise change these names into xray/tracing specific.